QuasarHelios2015/src/org/team1540/quasarhelios/DriveCode.java

```diff
@@ -1,12 +1,17 @@
 package org.team1540.quasarhelios;
 
-import ccre.channel.BooleanInput;
-import ccre.channel.FloatInput;
+import ccre.channel.*;
+import ccre.igneous.*;
 
 public class DriveCode {
 	public static FloatInput leftJoystickChannel;
 	public static FloatInput rightJoystickChannel;
 	public static BooleanInput octocanumShifting;
+	private static FloatOutput rightFrontMotor = Igneous.makeTalonMotor(1, Igneous.MOTOR_FORWARD, .1f);
+	private static FloatOutput leftFrontMotor = Igneous.makeTalonMotor(2, Igneous.MOTOR_FORWARD, .1f);
+	private static FloatOutput rightBackMotor = Igneous.makeTalonMotor(3, Igneous.MOTOR_FORWARD, .1f);
+	private static FloatOutput leftBackMotor = Igneous.makeTalonMotor(4, Igneous.MOTOR_FORWARD, .1f);
```

Comment:
Two comments - remember that you can combine the motors on the same side with `Mixing.combine` (for the normal drive mode, not mechanum) - and that half of the motors will need to be reversed (`Igneous.MOTOR_REVERSE`, I think.)